content/en/blog/helm-at-kubecon-eu-25.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Helm @ KubeCon + CloudNativeCon EU '25"
+slug: "helm-at-kubecon-eu-25"
+authorname: "Karen Chu"
+author: "@karenhchu"
+authorlink: "https://bsky.app/profile/karenchu.online"
+date: "2025-03-31"
+---
+
+It's that time of the year again – the Helm team is headed to KubeCon + CloudNativeCon EU '25 in London, UK this week! Helm 4 is in the works for later this year so be sure to join the conversation with our maintainers during our talk sessions and at our Helm booth in the Project Pavilion! See below for more details on all Helm-related activies throughout the week.
```

Comment:
`This week` is a "point in time" where it may be better to explicitly be descriptive on the dates. Just a suggestion if it makes sense